### ISSUE TITLE
Update libseccomp to 2.53+

### ISSUE THREAD

**Issue by tjkirch** [bottlerocket-os/bottlerocket#1815]
There's a conflict with systemd we'll have to figure out; this is what happens when trying to build systemd 247.10 after updating to libseccomp 2.53:

```
  #18 13.63 In file included from /x86_64-bottlerocket-linux-gnu/sys-root/usr/include/seccomp.h:821,                                                                        
  #18 13.63                  from ../src/shared/seccomp-util.c:6:                                                                                                           
  #18 13.63 ../src/shared/seccomp-util.c: In function 'seccomp_restrict_sxid':                                                                                              
  #18 13.63 ../src/shared/seccomp-util.c:2077:25: error: '__NR_openat2' undeclared (first use in this function); did you mean '__openat_2'?                                 
  #18 13.63  2077 |                         SCMP_SYS(openat2),                                                                                                              
  #18 13.63       |                         ^~~~~~~~
```

**Comment by cbgbt** [comment]
It looks like we'll want to include a version of systemd (or systemd + patches) which includes the fix from the issue linked above by @jlsalvador. Thanks for cross referencing!

Unfortunately our current version of systemd doesn't apply those patches cleanly. We'll probably want to table this until we have resolved #1671

**Comment by bcressey** [comment]
Going to pull this into the work for #1983.
